scripts/sql/delta/v1_9_0-v1_10_0/apolloconfigdb-v1_9_0-v1_10_0-after.sql

```diff
@@ -0,0 +1,79 @@
+--
+-- Copyright 2021 Apollo Authors
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+# delta schema to upgrade apollo config db from v1.9.0 to v1.10.0
+
+Use ApolloConfigDB;
```

Comment:
Apollo also has support for [flyway migration](https://github.com/apolloconfig/apollo/tree/master/scripts/flyway), is it possbile to add the schema change and data conversion to flyway?

Reply:
> Apollo also has support for [flyway migration](https://github.com/apolloconfig/apollo/tree/master/scripts/flyway), is it possible to add the schema change and data conversion to flyway?

Flyway is a good tool to manage database schemas. But currently, it makes duplicate workload in the project, could we drop this feature to make it simple? 

* scripts/sql
* scripts/flyway
* scripts/docker-quick-start/sql

I think it is unnecessary to maintain SQL in these 3 places at the same time.

Reply:
btw, the next release version is 2.0.0, so the SQL should be put into [v190-v200](https://github.com/apolloconfig/apollo/tree/master/scripts/sql/delta/v190-v200) directory.

> Flyway is a good tool to manage database schemas. But currently, it makes duplicate workload in the project, could we drop this feature to make it simple?
>* scripts/sql
>* scripts/flyway
>* scripts/docker-quick-start/sql
I think it is unnecessary to maintain SQL in these 3 places at the same time.

As we are releasing 2.0.0, which is a major version and it could introduce some big changes. So we could open a discussion thread to discuss whether to drop flyway support.

Reply:
Already moved it, and we discuss flyway after this release.